### PR TITLE
WRP-6258:Layout: add docs for `justify-content` CSS property

### PR DESCRIPTION
**Pull request by hoeunryuLG**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
We decided to add a prop to support gap . (ex. justify-content: space-between; ), but enact/ui/layout already has `align` prop to apply this. So we need more docs to guide it.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
add more docs for `align` prop of `Layout` and for `justify-content` CSS property.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-6258

### Comments
Enact-DCO-1.0-Signed-off-by: Hoeun Ryu <hoeun.ryu@lge.com>

### CODE REVIEW

File: packages/ui/Layout/Layout.js [hunk 1]
```diff
@@ -179,7 +179,27 @@ const LayoutBase = kind({
 		 * This includes support for `align-parts` which is shorthand for combining `align-items`
 		 * and `justify-content` into a single property, separated by a space, in that order.
 		 * This allows you to specify both the horizontal and vertical alignment in one property,
-		 * separated by a space.
+		 * separated by a space. For example, `align="center space-between"` means
```
Comment (seunghoh) [inline packages/ui/Layout/Layout.js:182]: Let's change the line starting from "For example," be sure to make a new line when build docs

File: packages/ui/Layout/Layout.js [hunk 2]
```diff
@@ -179,7 +179,27 @@ const LayoutBase = kind({
 		 * This includes support for `align-parts` which is shorthand for combining `align-items`
```
Comment (seunghoh) [inline packages/ui/Layout/Layout.js:179]: This includes support shorthand for combining

File: packages/ui/Layout/Layout.js [hunk 3]
```diff
@@ -179,7 +179,27 @@ const LayoutBase = kind({
...
 		 * and `justify-content` into a single property, separated by a space, in that order.
 		 * This allows you to specify both the horizontal and vertical alignment in one property,
-		 * separated by a space.
+		 * separated by a space. For example, `align="center space-between"` means
+		 * `align-items: center` and `justify-content: space-between` for each. `justify-content`
+		 * property can be used to align the flex items on the main axis and adjust gaps among the
```
Comment (seunghoh) [inline packages/ui/Layout/Layout.js:184]: flex items -> Cells

File: packages/ui/Layout/Layout.js [hunk 4]
```diff
@@ -179,7 +179,27 @@ const LayoutBase = kind({
...
+		 * flex items. To declare the `justify-content` property only, just add a heading space for
```
Comment (seunghoh) [inline packages/ui/Layout/Layout.js:185]: flex items -> Cells

**Review by seunghoh — CHANGES_REQUESTED** [review]

File: packages/ui/Layout/Layout.js (hunk 1 above)
Comment (hoeunryuLG) [reply packages/ui/Layout/Layout.js:182]: ok. I will fix it.

File: packages/ui/Layout/Layout.js (hunk 2 above)
Comment (hoeunryuLG) [reply packages/ui/Layout/Layout.js:179]: OK. I will fix it.

File: packages/ui/Layout/Layout.js (hunk 3 above)
Comment (hoeunryuLG) [reply packages/ui/Layout/Layout.js:184]: OK. I will fix it.

File: packages/ui/Layout/Layout.js (hunk 4 above)
Comment (hoeunryuLG) [reply packages/ui/Layout/Layout.js:185]: OK. I will fix it.

**Comment by hoeunryuLG** [comment]
I've pushed a fix-up patch to address the reviewer's comments.

**Review by seunghoh — APPROVED** [review]
LGTM

**Pull request merged**